Docs/sphinx/DeveloperGuide.rst

```diff
@@ -0,0 +1,48 @@
+.. highlight:: rst
+
+
+Developer Guidelines
+====================
+
+The following developer guidelines apply to all code that is to be committed to the `Pele` suite. 
+
+Before adding files for a commit to be pushed to a branch of `PelePhysics`, first run the following formatting tools depending on the type of code:
+
+
+C++ Code
+--------
+All C++ code should be processed by the Clang formatter prior to being added for commit.
+
+Run ``clang-format``::
+
+    clang-format -i .
```

Comment:
We should actually probably specify on which files `clang-format` should be run instead of doing the "." like I put here. I don't think this has the desired result unlike the python formatters which work well with the "." format.

Reply:
OK, I changed this